db/resources/bin/pg_backup.sh

```diff
@@ -116,6 +109,16 @@ function perform_backups()
 			fi
 		fi
 
+		# Assume at least 10MB for successful backup
+		MIN_BYTE_COUNT=10000000 
+
+		if [ $(stat -c%s $FINAL_BACKUP_DIR"$DATABASE".sql.gz) > MIN_BYTE_COUNT ]; then
```

Comment:
isn't it necessary to call it as variable like `$MIN_BYTE_COUNT` with Dollar-Sign before?

otherwise we would compare the size with a string value :thinking: